.dockerignore

```diff
@@ -1,4 +1,3 @@
-bin/*
```

Comment:
is this change intended?

Reply:
Yes! You might remember me losing my mind because the `ADD` command not working because of this dumb line.

Reply:
ups, it is dockerignore, not gitignore, then yes I remember 😅